### PR TITLE
[Develop] components Avatar, AvatarGroup

### PR DESCRIPTION
**Pull request by naimdanialjohar**
# Description

components Avatar and AvatarGroup

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings

### CODE REVIEW

File: src/Avatar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { Avatar as AntAvatar, AvatarProps as AntAvatarProps } from 'antd';
+
+import 'antd/lib/avatar/style/index.css';
```
Comment (AladdinNullptr) [inline src/Avatar/index.tsx:4]: move to style.css

File: src/Avatar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import './style.css';
+
+export type AvatarProps = AntAvatarProps;
+
+const Avatar: React.FC<AvatarProps> = props => {
+  return <AntAvatar {...props} />;
+};
+
+Avatar.defaultProps = {
+  size: 'default'
```
Comment (AladdinNullptr) [inline src/Avatar/index.tsx:14]: I don't think it is needed. You can verify and remove if not needed

File: src/Avatar/stories/index.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { default as Avatar } from '../index';
+import { AvatarProps } from 'antd';
+
```
Comment (AladdinNullptr) [inline src/Avatar/stories/index.stories.tsx:5]: See import error above

File: src/AvatarGroup/stories/index.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { default as AvatarGroup, AvatarGroupProps } from '../index';
+import Avatar from '../../Avatar';
+
```
Comment (AladdinNullptr) [inline src/AvatarGroup/stories/index.stories.tsx:6]: See import error above

File: src/AvatarGroup/index.tsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { Avatar as AntAvatar } from 'antd';
+
+import 'antd/lib/avatar/style/index.css';
```
Comment (AladdinNullptr) [inline src/AvatarGroup/index.tsx:4]: move to style.css

File: src/AvatarGroup/stories/index.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+export const Text = Template.bind({});
+
+Text.args = {
+  label: 'AvatarGroup',
+  children: (
+    <>
+      <Avatar>lala</Avatar>
+      <Avatar>lala</Avatar>
+      <Avatar>lala</Avatar>
```
Comment (AladdinNullptr) [inline src/AvatarGroup/stories/index.stories.tsx:33]: remove since it is not part of the story

File: src/AvatarGroup/stories/index.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+};
+
+export const Mixed = Template.bind({});
+
+Mixed.args = {
+  label: 'AvatarGroup',
+  children: (
+    <>
+      <Avatar src={'https://joeschmoe.io/api/v1/random'} />
+      <Avatar>lala</Avatar>
```
Comment (AladdinNullptr) [inline src/AvatarGroup/stories/index.stories.tsx:58]: remove

File: src/AvatarGroup/stories/index.stories.tsx [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+  component: AvatarGroup,
+  argTypes: {
+    size: {
+      control: {
+        type: 'radio',
+        options: [64, 'large', 'default', 'small']
+      },
+      defaultValue: 64
+    }
+  }
```
Comment (AladdinNullptr) [inline src/AvatarGroup/stories/index.stories.tsx:18]: Use the new argTypes structure I gave you. Did you make sure there are no new warnings?

**Review by AladdinNullptr — APPROVED** [review]

File: src/Avatar/index.tsx [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+import { Avatar as AntAvatar, AvatarProps as AntAvatarProps } from 'antd';
+import './style.css';
+
+export type AvatarProps = AntAvatarProps;
+
+const Avatar: React.FC<AvatarProps> = props => {
+  return <AntAvatar {...props} />;
+};
+
+Avatar.defaultProps = {};
```
Comment (alex-safian) [inline src/Avatar/index.tsx:11]: Remove this line

File: src/AvatarGroup/index.tsx [hunk 10]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Avatar as AntAvatar } from 'antd';
+import { GroupProps as AntAvatarGroupProps } from 'antd/lib/avatar';
+import './style.css';
```
Comment (alex-safian) [inline src/AvatarGroup/index.tsx:4]: Add a line break.

File: src/AvatarGroup/index.tsx [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+
+export type AvatarGroupProps = AntAvatarGroupProps;
+
+const AvatarGroup: React.FC<AvatarGroupProps> = props => {
+  return <AntAvatar.Group {...props} />;
+};
+
+AvatarGroup.defaultProps = {};
```
Comment (alex-safian) [inline src/AvatarGroup/index.tsx:12]: Remove this line

**Pull request merged**